t/alignment/plainmath.xml

```diff
@@ -66,8 +68,10 @@
               </XMCell>
               <XMCell rowspan="2">
                 <XMWrap>
-                  <XMTok depth="6.8pt" font="italic" height="0" role="CLOSE" yoffset="-6.8pt">)</XMTok>
-                  <XMTok font="italic" height="6.8pt" yoffset="-6.8pt"> </XMTok>
+                  <XMWrap class="unparsed">
```

Comment:
This is something I still want to fix - I'd like to reuse existing XMWrap elements rather than creating a new one. Let me see...

Reply:
Hm, on the other hand my check is happening at a point where the failure to parse here happens on the single token holding the `)`. The outer XMWrap is unrelated to the specific pass through MathParser. So maybe this is OK as it is.

Reply:
It's kinda unpleasant, but I've got to look at the parse logic a bit more closely to see if/where the `XMWrap` needs to be added.